### PR TITLE
vlc-video: Enable audio track selection

### PR DESCRIPTION
**Pull request by pkviet**
Adds an option for selecting input audio track.
See related mantis issue: https://obsproject.com/mantis/view.php?id=1407

### CODE REVIEW

File: plugins/vlc-video/vlc-video-source.c [hunk 1]
```diff
@@ -436,6 +438,11 @@ static void add_file(struct vlc_source *c, struct darray *array,
 					network_caching_option.array);
 			dstr_free(&network_caching_option);
 		}
+		struct dstr track_option = { 0 };
+		dstr_catf(&track_option,
+			":audio-track=%d", track_index - 1);
```
Comment (RytoEX) [inline plugins/vlc-video/vlc-video-source.c]: Style:  Tab twice to indent a line continuation (see line 434->436 above).

```suggestion
				":audio-track=%d", track_index - 1);
```

File: plugins/vlc-video/vlc-video-source.c [hunk 2]
```diff
@@ -489,7 +496,7 @@ static void vlcs_update(void *data, obs_data_t *settings)
 	obs_data_array_t *array;
 	const char *behavior;
 	size_t count;
-	int network_caching;
+	int network_caching, track_index;
```
Comment (RytoEX) [inline plugins/vlc-video/vlc-video-source.c]: Style:  Avoid declaring multiple variables on a single line.

```suggestion
	int network_caching;
	int track_index;
```

File: plugins/vlc-video/vlc-video-source.c [hunk 3]
```diff
@@ -540,13 +549,13 @@ static void vlcs_update(void *data, obs_data_t *settings)
...
 				add_file(c, &new_files.da, dir_path.array,
-						network_caching);
+						network_caching, track_index);
 			}
 
 			dstr_free(&dir_path);
 			os_closedir(dir);
 		} else {
-			add_file(c, &new_files.da, path, network_caching);
+			add_file(c, &new_files.da, path, network_caching, track_index);
```
Comment (RytoEX) [inline plugins/vlc-video/vlc-video-source.c]: Style:  Line exceeds 80 characters.

```suggestion
			add_file(c, &new_files.da, path, network_caching,
					track_index);
```

File: plugins/vlc-video/vlc-video-source.c (hunk 1 above)
Comment (pkviet) [reply plugins/vlc-video/vlc-video-source.c]: done

File: plugins/vlc-video/vlc-video-source.c (hunk 2 above)
Comment (pkviet) [reply plugins/vlc-video/vlc-video-source.c]: done

File: plugins/vlc-video/vlc-video-source.c (hunk 3 above)
Comment (pkviet) [reply plugins/vlc-video/vlc-video-source.c]: done

**Pull request merged**